src/main/java/org/osc/controller/nsc/restserver/SampleSdnServletDelegate.java

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ * Copyright (c) Intel Corporation
+ * Copyright (c) 2017
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package org.osc.controller.nsc.restserver;
+
+import static org.osgi.service.http.whiteboard.HttpWhiteboardConstants.*;
+
+import java.io.IOException;
+
+import javax.servlet.Servlet;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+
+import org.glassfish.jersey.server.ResourceConfig;
+import org.glassfish.jersey.server.ServerProperties;
+import org.glassfish.jersey.servlet.ServletContainer;
+import org.osc.controller.nsc.restserver.api.InspectionHookApis;
+import org.osc.controller.nsc.restserver.api.InspectionPortApis;
+import org.osc.controller.nsc.restserver.api.NetworkElementApis;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+import com.fasterxml.jackson.jaxrs.json.JacksonJaxbJsonProvider;
+
+@Component(name = "nsc.servlet", service = Servlet.class, property = {
+
+        HTTP_WHITEBOARD_SERVLET_NAME + "=" + "OSC-API", HTTP_WHITEBOARD_SERVLET_PATTERN + "=/testplugin/sdn/*",
```

Comment:
Can the URL be just nsc/*?

Also the Servlet name should reflect the nsc as well. Similar comment for the security manager api.

OSC-API should only be used for the core api. Whiteboard target i beleive needs to be OSC-API.

Correct me if i am wrong

Reply:
ok, will change the URL to nsc/* and also Servlet name to "NSC-API". I will adapt this as well for securitysample plugin.

Reply:
@arvindn05 the reason for having `testplugin` is to explicitly call out in the url this is a test entry point. Having simply `nsc` hides that. Also the plugin is really not nsc, but really a sample so I recommend not having nsc in the url name. For the manager the url is `/sm/` so what would the name be for consistency? ism?  The urls `/testplugin/sdn` and `/testplugin/sm` seem clear and consistent to me.  In the future we can also have /testplugin/sdn-sfc, for instance. @sudhirappaji  before you change this please share what will be the name in the url for both manager and sdn.

Reply:
makes sense. But instead of test plugin can we use consistent terminology maybe 

>sample/mgr/ism/*
>sample/sdn/nsc/*

Reply:
@sudhirappaji @arvindn05 the names above are good we can go with that.

Reply:
ok, will handle as proposed.